### PR TITLE
SEO-17 Per-wiki sitemap v1.1 - footer links on Main pages

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 1]
```diff
@@ -49,6 +47,7 @@ private function getGlobalFooterLinks() {
 				"Global Footer's links not found in messages",
 				[ 'exception' => new Exception() ]
 			);
+			wfProfileOut( __METHOD__ );
```
Comment (saipetch) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php]: duplicate wfProfileOut()?

Comment (gabrys) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php]: Oh, you're right. I noticed a return without wfProfileOut, but it seems there was one even before WikiaLogger.

File: extensions/wikia/GlobalFooter/templates/GlobalFooter_index.php [hunk 2]
```diff
@@ -10,11 +10,11 @@
 		</div>
 		<ul>
 			<?php
-			foreach ($footerLinks as $link):
+			foreach ( $footerLinks as $link ):
 				?>
 				<li>
 					<?php
-					if ($link['isLicense']) {
+					if ( isset( $link['isLicense']) ) {
```
Comment (saipetch) [inline extensions/wikia/GlobalFooter/templates/GlobalFooter_index.php]: missing a space close to ) ?

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 3]
```diff
@@ -57,12 +56,19 @@ private function getGlobalFooterLinks() {
 			$link = trim( $link );
 			if ( strpos( $link, '*' ) === 0 ) {
 				$parsedLink = parseItem( $link );
-				if ( ( strpos( $parsedLink['text'], 'LICENSE' ) !== false ) || $parsedLink['text'] == 'GFDL' ) {
-					$parsedLink['isLicense'] = true;
+
+				if ( strpos( $parsedLink['text'], 'SITEMAP' ) !== false ) {
+
```
Comment (saipetch) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php]: too many empty line?

File: extensions/wikia/GlobalFooter/GlobalFooterController.class.php [hunk 4]
```diff
@@ -57,12 +56,19 @@ private function getGlobalFooterLinks() {
...
+					$parsedLinks = array_merge( $parsedLinks, $this->generateSitemapLinks() );
```
Comment (saipetch) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php:60]: too many empty lines?
$parsedLinks[] = $this->generateSitemapLinks(); ?

Comment (gabrys) [inline extensions/wikia/GlobalFooter/GlobalFooterController.class.php:60]: Well, we don't want to overwrite the previous links already in parsedLinks, thus array_merge :-)

**Pull request merged**